portal/2023-08-04-changelog.md

```diff
@@ -0,0 +1,25 @@
+---
+authors: posdealer
+slug: portal/posdealerapi
+tags: [Portal, PosDealerAPI]
+---
+
+# PosDealer API Postman Collection Released!
+
+We're excited to launch our Postman Collection for the PosDealer API, designed to offer the most needed actions for our partners. This collection enables PosDealers to manage PosOperators, outlets, and product rollouts effortlessly.
+
+## What's Included?
+
+Sample requests to manage PosOperators, and outlets and perform product rollouts.
+
+## How to begin?
+
+Start by retrieving relevant data for your PosOperators. You may invite new PosOperators and manage their outlets.
+Utilize the check of available entitlements and use rollout plans to transfer entitlements.
+
+For those who prefer a graphical user interface, all operations are accessible in the fiskaltrust.Portal.
+
+## where to begin?
+
+Discover the convenience and usability of our [PosDealer API Postman Collection](https://posdealer-samples.docs.fiskaltrust.cloud/ "PosDealer API") now!  
+Your feedback is crucial in helping us improve. Reach out to us at [feedback+portal](feedback+portal@fiskaltrust.cloud).
```

Comment:
Correct email syntax:
[Contact Us](mailto:admin@w3schools.io)

Reply:
Damn it. Markdown needs to be escaped in the comments. Here the link instead https://www.w3schools.io/file/markdown-links/

Reply:
changed the link